src/shared/components/person/reports.tsx

```diff
@@ -135,8 +175,51 @@ export class Reports extends Component<ReportsRouteProps, ReportsState> {
   }
 
   render() {
+    const removePostForm = this.state.removePostForm;
+    const removeCommentForm = this.state.removeCommentForm;
+    const banFromCommunityForm = this.state.banFromCommunityForm;
+    const adminBanForm = this.state.adminBanForm;
     return (
       <div className="person-reports container-lg">
+        {removePostForm && (
+          <ModActionFormModal
```

Comment:
These modals should be inside the component, not the screen, just like they are in `post-listing` or `comment-node` with the dropdowns there.